### PR TITLE
add vendor extension for correct model generation

### PR DESCRIPTION
**Pull request by jvc9109**
This is a free-form object which was not properly generated due to the `aditionalProperties` parameter. Enforced the `x-no-generate` label and `x-is-free-form`.

### CODE REVIEW

File: openapi/components/schemas/HttpHeaders.yaml [hunk 1]
```diff
@@ -5,3 +5,5 @@ additionalProperties:
 example:
   Content-Type: "application/json"
   Accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8'
+x-no-generate: true
+x-is-free-form: true
```
Comment (justnero) [inline openapi/components/schemas/HttpHeaders.yaml:9]: Missing trailing new line

Comment (ashkarpetin) [inline openapi/components/schemas/HttpHeaders.yaml:9]: Please describe all new extensions in https://github.com/Rebilly/api-definitions/blob/main/specification-extensions.md

Comment (ashkarpetin) [reply openapi/components/schemas/HttpHeaders.yaml:9]: I have no idea what is the purpose of these extensions without proper description.

File: specification-extensions.md [hunk 2]
```diff
@@ -12,3 +12,18 @@ This document contains all OpenAPI extensions used in the Rebilly API definition
 * `x-product`
   This property declares operations that must be included into the specific API definition bundle.
   See more [here](plugins/products-bundler/README.md).
+
+* `x-is-free-form`
+  This flag is currently used for the php sdk autogeneration. It indicates that the schema should not be generated as a model.
```
Comment (noon-dawg) [inline specification-extensions.md:17]: nitpick:
```suggestion
  This flag is used for PHP SDK autogeneration. It indicates that the schema should not be generated as a model.
```

File: specification-extensions.md [hunk 3]
```diff
@@ -12,3 +12,18 @@ This document contains all OpenAPI extensions used in the Rebilly API definition
...
+
+* `x-enum-mapping`
+  This property enumerates the mapping options for a custom discriminator schema. For example `openapi/components/schema/ReadyToPay/Adjust/AdjustPaymentMethod.yaml`.
+
+* `x-no-generate`
+  This flag is used for the php sdk autogeneration. It indicates that the schema should not be generated. Any updates in schemas with this flag requires a manual update of the sdk.
```
Comment (noon-dawg) [inline specification-extensions.md:23]: nitpick:
```suggestion
  This flag is used for PHP SDK autogeneration. It indicates that the schema should not be generated. Updates to schemas with this flag require a manual update of the SDK.
```

File: specification-extensions.md [hunk 4]
```diff
@@ -12,3 +12,18 @@ This document contains all OpenAPI extensions used in the Rebilly API definition
...
+
+* `x-abstract-parent`
+  This property is used for the php sdk autogeneration. It indicates the parent abstract class in the case the generator is not able to properly define it.
+
+* `x-no-abstract-methods`
+  This flag is used for the php sdk autogeneration. It indicates that the generated model will not implement getters nor setters.
```
Comment (noon-dawg) [inline specification-extensions.md:29]: nitpick:
```suggestion
  This flag is used for PHP SDK autogeneration. It indicates that the generated model will not implement getters or setters.
```

**Review by noon-dawg — COMMENTED** [review]
very minor nitpicks

**Review by justnero — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**